helm/cluster-operator/templates/deployment.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: cluster-operator
-  namespace: {{ .Values.namespace }}
+  name: {{ tpl .Values.resource.default.name  . }}
+  namespace: {{ tpl .Values.resource.default.namespace  . }}
   labels:
-    app: cluster-operator
+    app: {{ tpl .Values.resource.default.name  . }}
```

Comment:
Should this also have the version label?

Reply:
Done.